Engine Hacks/Necessary/UnsignedHitRate/HIT_rate_unsingned_text.s

```diff
@@ -0,0 +1,65 @@
+.thumb
+.align
+
+.macro blh to, reg=r3
+  ldr \reg, =\to
+  mov lr, \reg
+  .short 0xf800
+.endm
+
+
+@Resend the breaking code.
+ldsh r3, [r0, r2] @pointer:0203A54C (BattleUnit@gBattleActor.battleHitRate )
+mov r0 ,r4
+mov r1, #0x24
+mov r2, r9
+
+Start:
+push {r4,r5,r6}
+mov r4 ,r0
+mov r5 ,r2
+mov r6 ,r3
+blh 0x08003e54   @Text_SetXCursor
+mov r0 ,r4
+mov r1 ,r5
+blh 0x08003e60   @Text_SetColorId
+mov r0 ,r4
+mov r1 ,r6
+bl Text_AppendNumberOr2Dashes_Unisngned
+pop {r4,r5,r6}
+
+ldr r3,=0x0801E98C+1
+bx r3
+
+Text_AppendNumberOr2Dashes_Unisngned:
```

Comment:
Why is this function exactly the same as the vanilla `Text_AppendNumberOr2Dashes`?